### ISSUE TITLE
@import statements are not parsed

### ISSUE THREAD

**Issue by idshdx** [elycruz/rollup-plugin-sass#38]
rollup-config.js:
```
export default {
  entry: 'src/main-aot.ts',
  dest: 'aot/dist/build.js', // outputs a single application bundle
  sourceMap: false,
  plugins: [
    sass({
      // Write all styles to the bundle destination where .js is replaced by .css
      output: true
    /*include: ['src/style/app.scss'],*/ this triggers many erros, syntax error at '@', syntax error at $header-gray: ... 

    }),
    nodeResolve({
      jsnext: true,
      main: true
    }),
.....
```

JS and CSS bundles are generated but where I have @import statements, the sass is not compiled into css.
colors.scss(which is imported in app.scss):
```
/*
OCEANA color palettes and usage
*/
/*TEXT*/
$header-gray: #303030; //Card header Text Card header icon
$blue-dark-grey: #304050; //main text
```
generated bundle.css(with no errors):
```
OCEANA color palettes and usage
*/
/*TEXT*/
/*UI*/
CODE FROM COLORS.SCSS SHOULD BE HERE BUT IS MISSING

.mat-elevation-z0 {
  box-shadow: 0px 0px 0px 0px rgba(0, 0, 0, 0.2), 0px 0px 0px 0px rgba(0, 0, 0, 0.14), 0px 0px 0px 0px rgba(0, 0, 0, 0.12); }
....
```
Are @import statements working?
If I'm missing something, pardon me.

**Comment by differui** [comment]
Hi @idzer0lis, Try to install latest package and build again.

And a few test cases about typescript were added:https://github.com/differui/rollup-plugin-sass/blob/master/test/index.js#L225

**Comment by arthurkhlghatyan** [comment]
same for me, what's the state of the issue?
![image](https://user-images.githubusercontent.com/7512048/31060300-584de850-a722-11e7-88a0-b0ffaa41db74.png)

**Comment by differui** [comment]
@arthurkhlghatyan Can you post your rollup config file?

**Comment by pjjanak** [comment]
I think the issue is that this plugin isn't treating `~` as a shortcut to node_modules. Whether that's actually a bug here or not is probably debatable.

**Comment by Alfredo-Delgado** [comment]
You'll need to pass an importer to node-sass like:

```js
sass({
  options: {
    importer(path) {
      return { file: path.replace(/^~/, 'node_modules/') };
    }
  }
})
```

**Comment by tomtobac** [comment]
Hi, I run into the same problem, but now with the `importer` option in not able to point out the correct path.

![image](https://user-images.githubusercontent.com/10562610/40553177-2cf92ad2-6042-11e8-9db6-b132da065c48.png)


Looks like the file that is looking is `node_modules/tcp-settings.` and it should be `node_modules/tcp-settings/index.scss`

**Comment by differui** [comment]
Hi @tomtobac try to use full path as the import id or setup your resolve plugin(rollup-plugin-node-resolve) with:

```js
resolve({
  extensions: ['.scss', '.sass'],
})
```

**Comment by differui** [comment]
fixed by #53.

**Comment by differui** [comment]
fixed by #53
